template/analytics.notify.version/body.html

```diff
@@ -77,17 +77,17 @@
       <div style="margin-top: 15px;color:#72767B;">
         <span>
           <span style="font-weight: 600;color:#363636;">Scan report</span> for GitHub repository
-          <a href="https://github.com/fabric8-ui/fabric8-ui" style="color: #0088ce; text-decoration: none;">https://github.com/fabric8-ui/fabric8-ui</a>
+          <a href={{.custom.repo_url}} style="color: #0088ce; text-decoration: none;">{{.custom.repo_url}}</a>
```

Comment:
@GeetikaBatra check is doubel_quotes is required or not.

Reply:
This will work.